### ISSUE TITLE
Logging for LinearClient

### ISSUE THREAD

**Issue by emorling** [tiagosiebler/bybit-api#171]
Is there a way to configure Logging for the LinearClient, similar to the Web Socket? 
There seems to be logging to the console for all requests per default.

**Comment by tiagosiebler** [comment]
Not at the moment, but I only see one console logger in the REST client and that feature is disabled by default anyway:
https://github.com/tiagosiebler/bybit-api/blob/master/src/util/BaseRestClient.ts#L426

At least in the latest release. Which version are you on? What kind of logs are you seeing? Are they "starting request" and "response" logs with a lot of network info?

**Comment by emorling** [comment]
Correct, there is a console.log in the BaseRestClient.js on line 99.

I am using version 2.3.2

**Comment by tiagosiebler** [comment]
I think there was one version where I had accidentally made the rookie error of including the network loggers in the release, but that was a long time ago. Looking at the base rest client for the previous release, I don't see any console logs there either (besides the time sync exception handler):
https://github.com/tiagosiebler/bybit-api/blob/84f9d618913f24dc9084e7a6650414ab29365d9f/src/util/BaseRestClient.ts

What kind of logs are you seeing?

Otherwise, if you want to try the upgrade route - the breaking changes to the latest release are extremely minimal. No methods or types were changed for the linear client, the only thing that changed is how you instance the linear client to provide api keys and other config options. One simple example:
https://github.com/tiagosiebler/bybit-api/blob/master/test/linear/private.read.test.ts#L12-L17

I've documented all the breaking changes here (some more changes around the ws client, although also relatively minimal):
https://github.com/tiagosiebler/bybit-api/releases/tag/v3.0.1

None of the tests are printing unexpected logs in the +300 API calls being made, so there shouldn't be an issue there.
